composer.json

```diff
@@ -13,8 +13,8 @@
     },
     "require": {
         "php": ">=5.3.3",
-        "zendframework/zend-config": "2.2.3",
-        "zendframework/zend-di": "2.2.3"
+        "zendframework/zend-config": "~2.3",
```

Comment:
sorry, i really need to stop committing so late at night, I can barely read :)

Yes, moved to ~2.2 which will allow from 2.2.X up to 3.0.0 (not inclusive)

Amended the commit